### ISSUE TITLE
Right click on a URL from the Preview tab opens both Context Menu and system browser.

### ISSUE THREAD

**Issue by AgentFire** [getinsomnia/insomnia#2839]
**Describe the bug**
Right click on a URL from the Preview tab opens both Context Menu and system browser.

**To Reproduce**
1. Receive any JSON which contains one or more valid HTTP(s) links.
2. Right click on a highlighted link, assuming to open a context menu.
3. Observe the context menu opens and then immediately a default system browser opens as well, navigating you to the link's URL.

**Expected behavior**
I expect only the Context Menu to pop up, and never the browser.

**Desktop (please complete the following information):**
Version: Insomnia Core 2020.4.2
Release date: 10/21/2020
OS: Windows_NT x64 10.0.19042
Electron: 9.1.1
Node: 12.14.1
V8: 8.3.110.13-electron.0
Architecture: x64
node-libcurl: libcurl/7.69.1-DEV OpenSSL/1.1.1d zlib/1.2.11 WinIDN libssh2/1.9.0_DEV nghttp2/1.40.0

**Comment by MrSnix** [comment]
I was **not able** to reproduce this issue on:
Version: Insomnia Core 2020.4.2
Release date: 24/11/2020
OS: Darwin x64 20.1.0
Electron: 9.1.1
Node: 12.14.1
V8: 8.3.110.13-electron.0
Architecture: x64
node-libcurl: libcurl/7.69.1 OpenSSL/1.1.1g zlib/1.2.11 brotli/1.0.7 libidn2/2.1.1 libssh2/1.9.0 nghttp2/1.41.0

I'll check on Windows too.

**EDIT**:
On Windows fresh install, i was **not able** to reproduce it:
Version: Insomnia Core 2020.4.2
Release date: 21/10/2020
OS: Windows_NT x64 10.0.10586
Electron: 9.1.1
Node: 12.14.1
V8: 8.3.110.13-electron.0
Architecture: x64
node-libcurl: libcurl/7.69.1-DEV OpenSSL/1.1.1d zlib/1.2.11 WinIDN libssh2/1.9.0_DEV nghttp2/1.40.0

**Comment by AgentFire** [comment]
Do you need me to record a video?

**Comment by MrSnix** [comment]
> Do you need me to record a video?

No, I trust you. I'm leaving here some details if someone wants to pick this up.
Sadly, I have no clue what's the cause.

**Comment by AgentFire** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

Bump, then.

**Comment by DuMuT6p** [comment]
Hello,
I noticed this issue also and it will be very useful if we can have it resolved.
